wiki/Tournaments/GSTLIVE/21/en.md

```diff
@@ -0,0 +1,159 @@
+---
+tags:
+  - GSTLIVE
+  - GST
+  - GST LIVE
+---
+
+# GST:LIVE 2021
+
+![GST:LIVE 2021](img/banner.jpg)
+
+**GST:LIVE 2021** was a 1v1 single-elimination osu! tournament hosted by ::{ flag=SG }:: [megumic](https://osu.ppy.sh/users/7537133). It was the first iteration of GST:LIVE, held on the 14 March 2021 at Bountie Arena.
+
+## Tournament schedule
+
+### Pre-event
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2021-02-25/2022-03-06 |
+
+### Day of the event (14 March 2021)
+
+| Event | Timestamp (UTC+08) |
+| --: | :-- |
+| Quarterfinals | 13:00 – 15:00 |
+| Semifinals | 15:00 – 16:20 |
+| Quarterfinals | 16:20 – 17:00 |
+
+## Prizes
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Phyiscal medal |
```

Comment:
in the future, when you're applying suggestions directly, [do it like this](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#applying-reviews).

now i'll have to check that everything was applied correctly (i.e. basically read through everything again) >.<

Reply:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Physical medal |
```